src/en/proposals/moony-better-ai.md

```diff
@@ -0,0 +1,74 @@
+
+The AI can additionally spend Compute (it's main resource) to enhance it's view in various ways, for example placing **tasks** down that perform various automation tasks (primarily in the form of "do x if y", for example "if camera detects motion, send alert"). Each task consumes a constant amount of Compute based on what it's trigger is (things like actively monitoring a camera are dramatically more expensive than only monitoring suit sensors, for example). This, and other utilities are the primary way the AI gains its omnipresence.
+
+## The core (figuratively and literally)
+The AI's most basic resource is Compute, some minimum of which is required for it to be alive to begin with. The bulk of the AI's compute traditionally comes from it's **core**, located within a dedicated high security structure attached to the station. The AI, however, is mobile, capable of offloading its computing processes onto most any machine with a screen at the cost of rendering that machine inoperable (for example, an APC providing compute will no longer power its room).
+
+The AI can spend compute on various things, for example the majority of its world views (for example, opening cameras, listening to radio, using machines, all overmap overlays, and generally just seeing things) all have a compute cost when enabled. The AI is fully capable of becoming very lightweight and running off just a few random machines instead of its cores, but doing so requires it to shut off the majority of its ways to view the world and additionally restricts most of its communication methods (including basic speech) This allows a crafty AI to survive total destruction of it's primary core area, given the crew don't get mad enough to kill it over their equipment no longer working in the meantime while it attempts to preserve its life.
+
+## The AI's role with the crew.
+The AI's role is that of both a manager and a servant, a lawed construct (the ONLY true lawed construct, the borgs are reduced to only being required to listen to the AI) that is in charge of managing the station's cyborgs/androids while following orders from the crew. **The AI is not just a door opener**, and lawsets should be chosen that do not obligate it to become such, in favor of allowing it to do what it does best (mass surveillence, automated tasks, organizing the borgs, etc). The AI additionally has limited resources when it comes to what tasks it can perform autonomously much of the AI's workload will be delegating both it's own tasks, the tasks of it's hands (the cyborgs), the requests of its hands (as they are mute per the Cool Borgs document), and the crew itself should they choose to listen to it. 
+
+## The Laws
```

Comment:
Happy to see people wanting to bring AI to '14, since it was the role I most enjoyed back when I played on '13 stations. I like that you want to evolve the role to fit on the new engine and not just copy over exactly what it was prior, so here's my 2 cents on AI laws. 

From my experience, AI laws were regularly a burden than an enhancement of roleplay/gameplay. Admin teams had to deal with a lot of headaches involving broken laws, and every station needed a 'Silicon policy' to explain what law interpretations were okay, but even those were subject to changes.

-

What if the concept of laws was advanced to the current understanding of AI? Instead of Asimov's Laws, we conceptualize AIs as NanoTrasen ChatGPTs? The 'Prompts' could work similarly to how Laws work, but no longer have a strict priority order, and thus more breathing room. 

Even if the same Asimov laws were kept as prompts, I'd assume that would be a big step to reducing AI/admin headaches over silicon policy.

Reply:
LLMs are not Actors, they do nothing when they have no input/prompt. Also, "replace the entire Law system" is a completely different approach from how we are doing Silicons and would immediately break multiple systems and game design. Imo it should be it's own proposal entirely, where it can address these issues